interfaces/apparmor/apparmor.go

```diff
@@ -37,46 +37,69 @@ import (
 	"github.com/snapcore/snapd/osutil"
 )
 
-// LoadProfile loads an apparmor profile from the given file.
+// LoadProfile loads apparmor profiles from the given files.
 //
 // If no such profile was previously loaded then it is simply added to the kernel.
 // If there was a profile with the same name before, that profile is replaced.
-func LoadProfile(fname string) error {
-	return loadProfile(fname, dirs.AppArmorCacheDir)
+func LoadProfiles(fnames []string) error {
+	return loadProfiles(fnames, dirs.AppArmorCacheDir)
 }
 
 // UnloadProfile removes the named profile from the running kernel.
 //
 // The operation is done with: apparmor_parser --remove $name
 // The binary cache file is removed from /var/cache/apparmor
-func UnloadProfile(name string) error {
-	return unloadProfile(name, dirs.AppArmorCacheDir)
+func UnloadProfiles(names []string) error {
+	return unloadProfiles(names, dirs.AppArmorCacheDir)
 }
 
-func loadProfile(fname, cacheDir string) error {
+func loadProfiles(fnames []string, cacheDir string) error {
+	if len(fnames) == 0 {
+		return nil
+	}
+
 	// Use no-expr-simplify since expr-simplify is actually slower on armhf (LP: #1383858)
-	args := []string{"--replace", "--write-cache", "-O", "no-expr-simplify", fmt.Sprintf("--cache-loc=%s", cacheDir)}
+	args := []string{"--replace", "--write-cache", "-O", "no-expr-simplify",
```

Comment:
No need to change this, golang has no 80 char line limit and we have long lines everywhere else in the code.

Reply:
IMO long lines are hard to read - newspapers and magazines have columns for a reason. But sure, I will revert this to attach to the project style.

Reply:
The change is still here. The code style in this project (and Go's community at large) does not adhere to the 80 columns limitation. Let's please revert the related changes in this PR.